### ISSUE TITLE
Is macOS Safari compatible?

### ISSUE THREAD

**Issue by savonarola27** [pyload/pyload-bookmarklet#3]
Hi, is thi bookmarklet compatible with macOS Safari? I tried to add a bookmark adding the bookmarklet code to the URL ma when I click on it nothing happens.

Instead works like a charm on FireFox so I think the problem in Safari.

Thanks.

**Comment by GammaC0de** [comment]
How do you add a bookmarklet  to Safari?

**Comment by savonarola27** [comment]
You can create a bookmarklet adding a bookmark for any page an then editing
it. In the url field you put the JavaScript code.

For an example of safari bookmarklet you can see this:

https://www.quixapp.com/browsers/safari/


Il giorno sab 1 giu 2019 alle 13:00 GammaC0de <notifications@github.com> ha
scritto:

**Comment by GammaC0de** [comment]
OK, it seems that the issue is safari's URL length limit..

**Comment by savonarola27** [comment]
Yes, in fact when I paste the code I can see a strange behavior. It seems
to delete some part and it is impossible to show all the pasted code.

I will open a issue to Apple.

Il giorno lun 3 giu 2019 alle 22:40 GammaC0de <notifications@github.com> ha
scritto:

**Comment by GammaC0de** [comment]
"I will open a issue to Apple." 
Good luck with that, hehe.. 

 

Luca Bomben<notifications@github.com> wrote:   Yes, in fact when I paste the code I can see a strange behavior. It seems
to delete some part and it is impossible to show all the pasted code.

I will open a issue to Apple.

Il giorno lun 3 giu 2019 alle 22:40 GammaC0de <notifications@github.com> ha
scritto:

**Comment by savonarola27** [comment]
Yeah. I know... there's no harm in trying :D


Il giorno mar 4 giu 2019 alle 09:18 GammaC0de <notifications@github.com> ha
scritto:
